sparse-file_test.go

```diff
@@ -79,18 +79,19 @@ func TestSparseFileRead(t *testing.T) {
 	require.NoError(t, err)
 	defer h.Close()
 
-	// Read a few randome ranges and compare to the expected blob content
-	for i := 0; i < 10; i++ {
-		start := rand.Intn(int(index.Length()))
+	// Read a few random ranges and compare to the expected blob content
+	for i := 0; i < 1000; i++ {
 		length := rand.Intn(int(index.Index.ChunkSizeMax))
+		offset := rand.Intn(int(index.Length()) - length)
```

Comment:
Shouldn't this be?
```go
offset := rand.Intn(int(index.Length()) - length - 1)
```

Reply:
Three hard things in computer science: off-by-one errors and naming things.

I'll fix it up.